docs/docs/firmware/project-structure/index.md

```diff
@@ -20,7 +20,9 @@ In this tutorial, we will __recreate the Blink project from scratch__ for multip
 
 ## Prepare the project folder
 
-All projects are stored under the `projects/` directory. They can be grouped into subfolders. For example, all demo projects are stored under `projects/Demo` and all EV5 firmware projects are in `projects/EV5`.
+All projects are stored under the `projects/` directory and each project has its own folder. For example, the `FrontController` project is stored at `projects/FrontController/`.
```

Comment:
This wording makes it seem like all projects must be directly under projects/. Can you work in the old wording about "can be grouped in sub folders?"